tests/install_upgrade_operators/crypto_policy/conftest.py

```
@@ -119,3 +136,149 @@ def services_to_check_connectivity(hco_namespace, admin_client):
+
+@pytest.fixture(scope="session")
+def enabled_aaq(admin_client, hco_namespace, hyperconverged_resource_scope_session):
+    with enabled_aaq_in_hco(
+        client=admin_client,
+        hco_namespace=hco_namespace,
+        hyperconverged_resource=hyperconverged_resource_scope_session,
+    ):
+        yield
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)